benchmarks/gc.py

```diff
@@ -13,11 +13,9 @@ def setup(self):
 
         self.gen("data", "cats_dogs")
         self.dvc("add", "data", "--quiet")
-        self.dvc("push", "data", "--quiet")
 
         # remove everything from the local system
         self.dvc("remove", "data.dvc")
-        self.dvc("gc", "-w")
```

Comment:
Does it make sense to have it here? Won't we be cleaning empty cache in that case?